lms/static/scripts/frontend_apps/components/ValidationMessage.js

```diff
@@ -0,0 +1,45 @@
+import { createElement } from 'preact';
+import { useEffect, useState } from 'preact/hooks';
+import classNames from 'classnames';
+import propTypes from 'prop-types';
+
+/**
+ * Shows a single validation error message that can be open or closed.
+ * A user can also close the message by clicking on it.
+ */
+
+export default function ValidationMessage({ message, open, onClose }) {
+  const [showError, setShowError] = useState(false);
+
+  useEffect(() => {
+    setShowError(open);
+  }, [open]);
+
+  /**
+   * Closes the validation error message and notifies parent
+   */
+  const closeValidationError = () => {
+    setShowError(false);
+    onClose();
```

Comment:
Important! The `onClose` property is not set as `required` as a `propType`, but this assumes it exists.